content/developer/howtos/rdtraining/E_unittest.rst

```diff
@@ -6,7 +6,7 @@ Advanced E: Python Unit Tests
 
 .. tip:: This tutorial assumes you followed the Core Training.
 
-  To do the exercise, fetch the branch 14.0-core from the repository XXX.
+  To do the exercise, fetch the branch {CURRENT_RELEASE}-core from the repository XXX.
```

Comment:
Did anybody convert the training to latest versions ?

Reply:
Not yet. I guess creating the branches will be enough for now.